go.mod

```diff
@@ -15,20 +15,20 @@ require (
 	github.com/aws/aws-sdk-go-v2/feature/s3/manager v1.15.15
 	github.com/aws/aws-sdk-go-v2/service/s3 v1.48.1
 	github.com/containerd/console v1.0.4
-	github.com/containerd/containerd v1.7.24
-	github.com/containerd/containerd/api v1.7.19
+	github.com/containerd/containerd/api v1.8.0
+	github.com/containerd/containerd/v2 v2.0.1
 	github.com/containerd/continuity v0.4.5
-	github.com/containerd/errdefs v0.3.0
-	github.com/containerd/fuse-overlayfs-snapshotter v1.0.8
-	github.com/containerd/go-cni v1.1.10
+	github.com/containerd/errdefs v1.0.0
+	github.com/containerd/fuse-overlayfs-snapshotter/v2 v2.1.0
+	github.com/containerd/go-cni v1.1.11
 	github.com/containerd/go-runc v1.1.0
 	github.com/containerd/log v0.1.0
 	github.com/containerd/nydus-snapshotter v0.14.0
```

Comment:
Looks like nydus v0.15 is needed. Current brings in duplicate version of archive pkg etc.